cluster/gce/configure-vm.sh

```diff
@@ -661,7 +688,17 @@ if [[ -z "${is_push}" ]]; then
   remove-docker-artifacts
   run-salt
   set-good-motd
+
+  curl-metadata k8s-user-startup-script > ${INSTALL_DIR}/k8s-user-script.sh
+  user_script=$(cat ${INSTALL_DIR}/k8s-user-script.sh)
+  if [[ ! -z ${user_script} ]]; then
+    wait-for-cluster-healthy-or-bust
+    chmod u+x ${INSTALL_DIR}/k8s-user-script.sh
+    echo "== running user startup script =="
+    ${INSTALL_DIR}/k8s-user-script.sh
```

Comment:
So that it can support `#!python`, etc scripts and also so that exit codes don't kill this script.